File: CodeChef/Rating1000to1200/AProblemonSticks/mySolution.py
t=int(input())
while t>0 : 
    t-= 1 
    n=int(input())
    v = list(map(int, input().split()))
    v=set(v)
    if 0 in v:
        c=len(v)-1
    else:
        c=len(v)
    print(c)

File: CodeChef/Ranking1000to1200/ChefandDolls/mySolution.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner input = new Scanner(System.in);
		int t=input.nextInt();
		while(t-->0)
		{
		    int n=input.nextInt();
		    int[] v=new int[n+1];
		    v[0]=100001;
		    for(int i=0; i<n; i++)
		       v[i+1]=input.nextInt();
		    Arrays.sort(v);
		    for(int i=0;  i<n; i+=2)
		    {
		        if(v[i]!=v[i+1])
		        {
		           System.out.println(v[i]);
		           break;
		        }
		    }
		}
	}
}
